### DISCUSSION TITLE
Help me restore app-tabs in side panel

### DISCUSSION THREAD

**Discussion by AndreyChursin** [alefragnani/vscode-bookmarks#662] [Q&A] [answered]
Help me please, I couldn't find this EXPLORER tab:
<img width="517" alt="image" src="https://github.com/alefragnani/vscode-bookmarks/assets/11028058/899e81fe-a573-4a47-bfed-c3dd98e62449">

And I don't know how to restore this one in my side-panel...
<img width="422" alt="image" src="https://github.com/alefragnani/vscode-bookmarks/assets/11028058/e75e8478-0d77-465e-ad66-453db79f48ab">

**Comment by dg00GT00** [comment]
Hi @AndreyChursin,
If you click on "View and More Actions..." aka "..." on the the top of bookmarks side panel you will see an option to reselect the bookmark dropdown menu.

**Comment by alefragnani** [comment, marked as answer]
Hi @AndreyChursin ,

First of all, sorry for the late reply.

If you are still facing this issue, try using the `Bookmarks: Focus on Explorer View` command to find out where it is. Those `Focus on XXX View` commands are automatically created by VS Code for every view that an extension provides.

If it does not work, you could try another _hardcore_ built in command `View: Reset View Locations`. As it name says, it will _reset_ the state/location of every view, returning them to the default location.

Hope this helps
